eventuals/event-loop.h

```diff
@@ -676,6 +682,9 @@ class EventLoop final : public Scheduler {
       template <typename Arg>
       using ValueFrom = void;
 
+      template <typename Arg, typename Errors>
+      using ErrorsFrom = tuple_types_union_t<Errors, std::runtime_error>;
```

Comment:
Hmm, this shouldn't compile ... so it must not be getting used? You need to wrap `std::runtime_error` in a `std::tuple`. Please validate and add a `static_assert` in one of the tests to check this one, thanks!